### DISCUSSION TITLE
LogContext: Display more than 10 rows by default

### DISCUSSION THREAD

**Discussion by svennergr** [grafana/grafana#56090] [Observability]
Currently the LogContext is showing 10 rows before and after the initial logline. That number was picked arbitrary and could be higher. I am happy to receive feedback regarding a good number.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
